packages/block-library/src/template-part/edit/selection/template-part-previews.js

```diff
@@ -202,20 +202,14 @@ export default function TemplatePartPreviews( {
 } ) {
 	const composite = useCompositeState();
 	const templateParts = useSelect( ( select ) => {
-		const publishedTemplateParts =
-			select( 'core' ).getEntityRecords( 'postType', 'wp_template_part', {
-				status: [ 'publish' ],
-				per_page: -1,
-			} ) || [];
-
 		const currentTheme = select( 'core' ).getCurrentTheme()?.stylesheet;
-		const themeTemplateParts =
+		return (
 			select( 'core' ).getEntityRecords( 'postType', 'wp_template_part', {
-				theme: currentTheme,
-				status: [ 'auto-draft' ],
+				theme: [ currentTheme, '_wp_is_original' ],
```

Comment:
Why does it all go into a single array?

Reply:
They are stored in the same taxonomy.

Reply:
What if a theme is called `_wp_is_original` or `_wp_file_based`